bin/cli.js

```diff
@@ -148,7 +149,24 @@ const setRequestHeaders = () => {
   addUserAgentHeader('HubSpot CLI', pkg.version);
 };
 
+const NO_CONFIG_VALIDATION = {
+  init: { skip: true },
+  auth: { skip: true },
+};
+
 const loadConfigMiddleware = async options => {
+  const maybeValidateConfig = () => {
+    const shouldValidate = options._.every(
+      command =>
+        !(NO_CONFIG_VALIDATION[command] && NO_CONFIG_VALIDATION[command].skip)
+    );
+    if (shouldValidate) {
+      if (!validateConfig()) {
```

Comment:
Nit: these can merged to `if (shouldValidate && !validateConfg())`

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1307/commits/c3fd8cc35cb4f896afc9f9af520107a6cd95732f